pkg/rpcclient/rpc.go

```diff
@@ -1285,3 +1285,44 @@ func (c *Client) TerminateSession(sessionID uuid.UUID) (bool, error) {
 
 	return resp, nil
 }
+
+// GetRawNotaryTransaction  returns main or fallback transaction from the
+// RPC node's notary request pool. This function uses a `[]bytes` result
+// from `getrawnotarytransaction`.
+func (c *Client) GetRawNotaryTransaction(hash util.Uint256) (*transaction.Transaction, error) {
+	var (
+		params = []any{hash.StringLE()}
+		resp   []byte
+		err    error
+	)
+	if err = c.performRequest("getrawnotarytransaction", params, resp); err != nil {
+		return nil, err
+	}
+	return transaction.NewTransactionFromBytes(resp)
+}
+
+// GetRawNotaryTransactionVerbose returns main or fallback transaction from the
+// RPC node's notary request pool. This function uses verbose
+// (*transaction.Transaction) result from `getrawnotarytransaction`.
```

Comment:
Ditto, make it unified with `GetRawTransactionVerbose` and let's add a comment about transaction hash and size.

Reply:
Sorry, didn't catch what about hash and size?

Reply:
Fixed.